src/Azure/AzureAD/Authentication.AzureAD.UI/src/AzureADAuthenticationBuilderExtensions.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.AspNetCore.Authentication
     /// <summary>
     /// Extension methods to add Azure Active Directory Authentication to your application.
     /// </summary>
+    [Obsolete("This is obsolete and will be removed in a future version. Use Microsoft.Identity.Web instead")]
```

Comment:
Do you want to provide an aka.ms link?
https://aka.ms/ms-identity-web ?

Reply:
(same for all the obsolete attribute. We've found, in Identity, that this was helping people migrating more easily.

Reply:
Perfect! I didn't know of the aka.ms link.